libs/tcp_wrappers/Makefile

```diff
@@ -30,7 +30,7 @@ define Package/libwrap
   MAINTAINER:=Peter Wagner <tripolar@gmx.at>
 endef
 
-TARGET_CFLAGS += $(FPIC)
+TARGET_CFLAGS += $(FPIC) -Wall
```

Comment:
I actually don't like this. It slows compilation down. Can you remove it?

Reply:
I'd prefer not too, and I'm not seeing any regression in compilation time, when running:

`time make -j1 package/tcp_wrappers/{clean,compile} V=s`

master (`abfd6d81e`):

```
real    0m36.094s
user    0m14.344s
sys     0m21.844s
```
tcp_wrappers-compilation-warnings (`869112e36`):

```
real    0m37.041s
user    0m14.828s
sys     0m21.984s
```

Reply:
Well, slows down by 1 second :). My view on -Wall is that it should be used by people developing the package, not necessarily people that compile or use it.

In any case, it just creates noise. This package has not seen updates in over 20 years. At this point, I think it's a better idea to phase it out gradually from dependent packages.

Reply:
> I actually don't like this. It slows compilation down. Can you remove it?

It shouldn't significantly.

> My view on -Wall is that it should be used by people developing the package, not necessarily people that compile or use it.

The people developing the package likely haven't tested extensively in a lot of cross-build environments such as our own, and hence regressions can creep in at any time with new releases.

One second of machine time is worth potential hours of someone having to painstakingly search for breakage where there previously was none.  Machine time is cheap; human time is precious.

Reply:
Hrm OK.

Reply:
> At this point, I think it's a better idea to phase it out gradually from dependent packages.

That, on the other hand, I will whole heartedly agree with.

I tried to send Wietse some improvements to `tcp_wrappers` (to support ranges, not just prefixes) about 12 years ago and he spanked for me still using it, saying I should rely on `iptables` to do what `tcp_wrappers` had been doing (!!!).

He's correct, of course.  Relying on a file which can easily be tampered with is insecure (yes, I know the file should only be writable by `root`).

But as you add more defense-in-depth, like `chroot`ing or docker, etc. then `iptables` becomes harder to subvert, whereas `tcp_wrappers` continues to have the same vulnerabilities.